packages/js/product-editor/src/index.ts

```diff
@@ -2,3 +2,25 @@ export {
 	ProductSectionLayout as __experimentalProductSectionLayout,
 	ProductFieldSection as __experimentalProductFieldSection,
 } from './product-section-layout';
+
+/**
+ * Utils
+ */
+export { formatCurrencyDisplayValue } from './utils/format-currency-display-value';
```

Comment:
Likely subjective, but an alternative here would be to add an `index.ts` file within the `/utils` directory to export all utilities, and then potentially just `export * from './utils'`. Makes it a little smoother to import these utils within the package itself, since you can import all from `../utils`.